enricher/api/src/main/java/io/fabric8/maven/enricher/api/BaseEnricher.java

```diff
@@ -59,6 +60,12 @@ public BaseEnricher(EnricherContext enricherContext, String name) {
         this.name = name;
     }
 
+    protected boolean useDeploymentforOpenShift() {
```

Comment:
Do we really need this method? Can't we simply use getValueFromConfig directly?

Reply:
I checked for getValueFromConfig method. That method is specific is not a general utility, but a specific method used in VertxHealthCheckEnricher.

Reply:
what? No. I think I wrote it to be used by all enrichers